src/components/programmingCard.js

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from "react"
+import { Link } from "gatsby"
+import "./programmingCard.scss"
+
+export default function ProgrammingCard({ title, summary, id, entries }) {
+  const href = `https://www.artsmatter.org/programs/${id}`
+  const [expanded, setExpanded] = useState(false)
+
+  const card = (title, summary, id) => {
+    return (
+      <div className="programming-card">
+        {title && <h4 className="programming-card__title">{title}</h4>}
+        {summary && <p className="programming-card__summary">{summary}</p>}
+        {id && (
+          <Link
+            className="programming-card__link"
+            to={`/programs/${id}`}
+            aria-label="Learn More"
+          >
+            Learn more
+          </Link>
+        )}
+      </div>
+    )
+  }
+
+  const expandableCard = (title, summary, nestedCards) => {
+    return (
+      <div
+        className={`programming-card ${expanded ? "is-open" : ""}`}
+        onClick={() => setExpanded(!expanded)}
+      >
+        {title && <h4 className="programming-card__title">{title}</h4>}
+        {summary && <p className="programming-card__summary">{summary}</p>}
+        <button
+          className="programming-card__expand-button"
+          aria-pressed={expanded}
+          aria-expanded={expanded}
+        >
+          <div className="programming-card__expand-icon"></div>
+        </button>
+        {nestedCards.length > 0 && (
+          <div
```

Comment:
Can we add an `aria-hidden` attribute here to hide it from screen readers when collapsed?